source/classes.tex

```diff
@@ -6901,12 +6984,15 @@
 \end{note}
 \end{itemize}
 
-\rSec2[class.rel]{Relational operators}
+\rSec2[class.compare.secondary]{Secondary comparison operators}
 \indextext{operator!relational!defaulted}%
 
 \pnum
-A defaulted relational operator function\iref{over.binary}
-for some operator \tcode{@}
+\indextext{operator!comparison!secondary}%
+A \defn{secondary comparison operator} is
```

Comment:
drop the \indextext and just say \defnadj{secondary}{comparison operator}

Reply:
I went with `\defnadj{secondary}{operator!comparison}` since we don't have a top level index entry for "comparison operator".

Reply:
I don't think "!" renders correctly in the \defnadj. You need to do a manual \defnx if you want to get clever.